kubecf-build-pipelines/external-releases/external-releases.yml

```diff
@@ -59,6 +59,6 @@ releases:
 #   sha1: 706218d94e13782d65daf926279adcd41ef3911a
 # TODO: replace url once final release is available.
 - name: sync-integration-tests-release
-  url: https://s3-us-west-2.amazonaws.com/dummy-releases/sync-integration-tests-release-v0.0.2.tgz
-  version: v0.0.2
-  sha1: 6d2d70efac87ee088fa3c6e8f886b65a7ccb0404
+  url: https://s3.amazonaws.com/suse-final-releases/sync-integration-tests-release-v0.0.3.tgz
```

Comment:
Unrelated change. Could you open another PR for this?

Reply:
The change was already merged in develop. Re-based the branch.